static/js/controllers/inbox.js

```diff
@@ -799,16 +798,23 @@ var feedback = require('../modules/feedback'),
         if (window.applicationCache.status === window.applicationCache.UPDATEREADY) {
           showUpdateReady();
         }
-        WatchDesignDoc(function() {
-          // if the manifest hasn't changed, prompt user to reload settings
-          window.applicationCache.addEventListener('noupdate', showUpdateReady);
-          // check if the manifest has changed. if it has, download and prompt
-          try {
-            window.applicationCache.update();
-          } catch(e) {
-            // chrome incognito mode active
-            $log.error('Error updating the appcache.', e);
-            showUpdateReady();
+        Changes({
```

Comment:
The WatchDesignDoc service created a changes feed against the server to watch for updates to the "medic" ddoc.

The Changes service creates one changes feed to the local pouchdb then looks through all registered callers for which to notify. This works here because the "medic-client" ddoc is replicated through the standard replication mechanism so we don't have to have a specific watcher for it.